src/main/scala/io/iohk/ethereum/db/storage/ReferenceCountNodeStorage.scala

```diff
@@ -28,18 +28,22 @@ import io.iohk.ethereum.utils.Logger
   * Storing snapshot info this way allows for easy construction of snapshot key (based on a block number
   * and number of snapshots) and therefore, fast access to each snapshot individually.
   */
-class ReferenceCountNodeStorage(nodeStorage: NodesStorage, blockNumber: Option[BigInt] = None)
+class ReferenceCountNodeStorage(nodeStorage: NodesStorage,
+                                blockNumber: Option[BigInt] = None)
   extends NodesKeyValueStorage {
 
   import ReferenceCountNodeStorage._
 
+  private val deathRowKey = drRowKey(blockNumber.get)
```

Comment:
I don't think this is safe, to call get here

Reply:
yep, you are right my bad. it should have been in `update` method.